src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/TornadoService.java

```diff
@@ -132,6 +133,7 @@ private void buildListingInstruction(HttpURLConnection conn, ListingData listing
             sb = ListingHelper.buildListingDocumentContent(listingData, tornadoConnection.getAccessKey(),
                     documentName, userDetails, caseType);
         }
+        log.info("In Tornado Service - Content of Document to generate: " + sb);
```

Comment:
What will this output to the logs?

Reply:
It is the json representation of data that get fed to template. I only need that for dev. I take it out.